app/data_definitions/two_column.rb

```diff
@@ -77,11 +77,11 @@ class TwoColumn < DataDefinitions::Base
 
       v201611: {
         slider: {
-          header: 'Campus Life \n at \n Chapman University',
+          header: 'Campus Life at Chapman University',
 
           slides: [
             {
-              subTitle: '',
+              subTitle: 'Subtitle Here',
```

Comment:
Changes look good. Only these last two here concern me. Restore these and if there are causing issues, they should be worked out with product owner.